### ISSUE TITLE
It is impossible to access Upload form with keyboard

### ISSUE THREAD

**Issue by CKFinderBOT** [ckfinder/ckfinder#254]
The "old" upload form cannot be accessed with keyboard, it's not included in tab order.

**Comment by CKFinderBOT** [comment]
Fixed. The updated code will be publicly available in the upcoming release.
